src/components/homePage/homePage.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import { useNavigate } from 'react-router-dom';
+import './homePage.css';
+import { BsInfoCircle } from 'react-icons/bs';
+
+const HomePage = (props) => {
+  const [search, setSearch] = useState('');
+  const navigate = useNavigate();
+  const { innerwidth } = props;
+  const { allData } = useSelector((state) => state.allCoins);
+
+  return (
+    <div className="homePage">
+      <form className="homePage__search">
+        <input
+          type="text"
+          placeholder="Search"
+          value={search}
+          onChange={(e) => setSearch(e.target.value)}
+        />
+
+        {' '}
+      </form>
+      <div className="homePage__products">
+        {allData
+          .filter((product) => product.name.toLowerCase().includes(search.toLowerCase()))
+          .map((product, index) => (
+            <div
+              className="homePage__product"
+              key={product.id}
+              style={index % 2 === 0 ? { backgroundColor: '#0096c7' } : { backgroundColor: '#00b4d8' }}
+            >
+              {innerwidth > 500 ? (
+                <img
```

Comment:
- [ ] Great work on the app, however it is a core requirement that your design matches the given design, this includes the layout meaning `composition and space between elements`. and typography `meaning the font weight of some elements, font face, font size e.t.c`. Kindly implement this

![image](https://user-images.githubusercontent.com/81932478/212000016-b7e1d840-a583-4706-93d0-68c8bda5dc83.png)

Your design
![image](https://user-images.githubusercontent.com/81932478/212000136-6a0ac0dd-001b-435f-b9fd-ede9faf5e059.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/212000375-f2ecf52f-5ea3-4f8f-b77b-511a8a8f4b8a.png)